### PR TITLE
Updated web.dev/trust-tokens

### PR DESCRIPTION
**Pull request by samdutton**
• Miscellaneous updates to wording and code samples.
• Added warning about TrustTokenV3 update.

Requires eng review/approval before merging.

@dvorak42 @davidvancleve fyi

### CODE REVIEW

File: src/site/content/en/blog/trust-tokens/index.md [hunk 1]
```diff
@@ -42,50 +58,44 @@ in the Chrome DevTools **Network** and **Application** tabs.
 </figure>
 
 {% Aside %}
-The Privacy Sandbox is a series of proposals to satisfy third-party use cases
-without third-party cookies or other tracking mechanisms. See
-[Digging into the Privacy Sandbox](/digging-into-the-privacy-sandbox)
+The Privacy Sandbox is a series of proposals to satisfy third-party use cases without third-party }
```
Comment (mihajlija) [inline src/site/content/en/blog/trust-tokens/index.md:61]:
```suggestion
The Privacy Sandbox is a series of proposals to satisfy third-party use cases without third-party 
```

File: src/site/content/en/blog/trust-tokens/index.md [hunk 2]
```diff
@@ -114,83 +124,91 @@ The following is adapted from
...
 {% Details %}
 {% DetailsSummary %}
 How can a website work out whether to trust you?
 {% endDetailsSummary %}
-You might have shopping history with an ecommerce site, checkins on a location
-platform, or account history at a bank. Issuers might also look at other factors
-such as how long you've had an account, or other interactions (such as CAPTCHAs
-or form submission) that increase the issuer's trust in the likelihood that
-you're a real human.
+You might have shopping history with an ecommerce site, checkins on a location platform, or account
```
Comment (mihajlija) [inline src/site/content/en/blog/trust-tokens/index.md:201]:
```suggestion
You might have shopping history with an e-commerce site, check-ins on a location platform, or account 
```

File: src/site/content/en/blog/trust-tokens/index.md [hunk 3]
```diff
@@ -238,59 +255,54 @@ fetch('issuer.example/.well-known/trust-token', {
...
+tokens will run out over time, since every redemption is sent to the same token issuer, which can 
+verify that each token is used only once. To mitigate risk, issuers could also sign fewer tokens.
 
 ### Request mechanisms
 
-It might be possible to allow for sending RRs outside of `fetch()`, for example
-with navigation requests. Sites might also be able to include issuer data in
-HTTP response headers to enable token redemption in parallel with page
-loading.
+It might be possible to allow for sending RRs outside of `fetch()`, for example with navigation
```
Comment (mihajlija) [inline src/site/content/en/blog/trust-tokens/index.md:303]: Does RR stand for redemption request?

**Review by mihajlija — APPROVED** [review]

File: src/site/content/en/blog/trust-tokens/index.md (hunk 3 above)
Comment (samdutton) [reply src/site/content/en/blog/trust-tokens/index.md]: Oops — good spot. Changed to 'redemption record'.

**Comment by dvorak42** [comment]
Sorry was OOO end of last week, overall looks good, but I'd actually drop the ".well-known" from all the paths, currently that's not required and its likely we won't end up having this under .well-known.

**Comment by samdutton** [comment]
Thanks Steven — I've removed .well-known.

**Comment by samdutton** [comment]
Meant to say — if no objections, I'll merge this.

**Comment by dvorak42** [comment]
LGTM.

**Comment by samdutton** [comment]
> LGTM.

Thanks Steven. 

Merged.

**Comment by davidvancleve** [comment]
Belated LGTM.

Thanks for this!

**Comment by samdutton** [comment]
Thanks David!

**Pull request merged**